### ISSUE TITLE
chrome is not allowed to use the microphone

### ISSUE THREAD

**Issue by Ogaiht10** [sdkcarlos/artyom.js#47]
I have a project that worked until months ago, but suddenly the chromer started not giving permission to use the microphone. I entered the chromer configuration to see if the url was in the blocked microphone addresses, but it was not. Can anyone help?

**Comment by webleyson** [comment]
One thing I noticed that caused issues was having artyom running in another window at the same time. Could that be an issue?

**Comment by sdkcarlos** [comment]
@webleyson  Indeed, **you can't** run 2 instances of the WebkitSpeechRecognition API in the browser. However in older versions of Chrome, this made the browser crash so I though it would evident, however i'll add it to the FAQ.

**Comment by sdkcarlos** [comment]
@Ogaiht10 , does the homepage of Artyom work or is only in your project ? Any error in the console?

**Comment by Ogaiht10** [comment]
This is only happening in my project. In the homepage of Artyom is working normal.
I'm not running two instances.
My project is: http://databindsolucoes.com.br/robo/robo.html

**Comment by Ogaiht10** [comment]
Did you see the project? Does anyone know what can it be?

**Comment by maximeburri** [comment]
Hello,
Same problem with our project : http://129.194.185.235:8080/touchpad-voice/345.

This problem occurs with an IP public or an IP in a private network, but it works fine when it's running on localhost.
This problem doesn't occur with old chrome version (like 58.0) but with recent (like 68.0).

![image](https://user-images.githubusercontent.com/5551924/32438788-1970dd00-c2ec-11e7-94e6-ccda0e27fd5e.png)

The cross in logo of microphone stays even if we authorize.

**Comment by Ogaiht10** [comment]
@sdkcarlos, is there any solution to our problem?

**Comment by maximeburri** [comment]
Any ideas ?

**Comment by maximeburri** [comment]
Oh. It's cause by chrome who block access on insecure origins with Chrome 60 from 25th of July ([Article : Access No Longer Works On Insecure Origins](https://addpipe.com/blog/microphone-camera-access-no-longer-works-insecure-origins/)).

We need to configure a secure origins (https) to resolve this issue.

**Comment by nilekshdhimer** [comment]
I have the same issue in localhost as well as on my website. My browser version is: **Version 69.0.3497.100 (Official Build) (64-bit)**
![2018-10-08_13h13_28](https://user-images.githubusercontent.com/7030318/46597022-0b99d380-cafc-11e8-99ac-6d99a0d5ce9c.png)

**Comment by nilekshdhimer** [comment]
@sdkcarlos  Hello, please give a solution for this.

**Comment by vietdq1291** [comment]
using SSL for your site and allow your site access microphone

**Comment by nilekshdhimer** [comment]
> using SSL for your site and allow your site access microphone

I have already allowed my site to access microphone

**Comment by vietdq1291** [comment]
oh i'm sorry i don't using this library i'm using webkitSpeechRecognition with angular 6

**Comment by howtostuff** [comment]
I currently have the same problem in a chrome extension that I am building.
For some reason, this does not show up on my PC but then I tried it on 2 different macs and on both this problem shows up.
